### PR TITLE
fix: fix mAPI Post status code transform

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-7731

## Description

Steps to reproduce:

Create a V4 message API with Get and Post entrypoints and Kafka endpoint

Apply Status Transformation Policy on response 

Configure the policy to transform 200 status and 202 into another value 

Send a Get request and a Post request

Check if the statuses were transformed as configured in the policy

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.0.1-APIM-7731-fix-mAPI-POST-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-transform-status-code/1.0.1-APIM-7731-fix-mAPI-POST-SNAPSHOT/gravitee-policy-transform-status-code-1.0.1-APIM-7731-fix-mAPI-POST-SNAPSHOT.zip)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
     <groupId>io.gravitee.policy</groupId>
     <artifactId>gravitee-policy-transform-status-code</artifactId>
-    <version>1.0.0</version>
+    <version>1.0.0-alpha.1</version>
```
Comment (phiz71) [inline pom.xml:33]: you can revert this change, the version will be  handled automatically

Comment (illichso) [reply pom.xml]: Done

**Review by phiz71 — APPROVED** [review]
